monitoring/rid_qualifier/flight_track_factory.py

```diff
@@ -0,0 +1,178 @@
+
+        cell_size_x = (self.maxx - self.minx)/(3) # create three columns
+        cell_size_y = (self.maxy - self.miny)/(2) # create two rows
+        grid_cells = [] 
+        for u0 in range(0, 3):  # 3 columns           
+            x0 =  self.minx + (u0 * cell_size_x)            
+            for v0 in range(0,2): # 2 rows
+                y0 = self.miny + (v0 *cell_size_y)
+                x1 = x0 + cell_size_x
+                y1 = y0 + cell_size_y
+                grid_cells.append(shapely.geometry.box(x0, y0, x1, y1))                
+        
+        self.flight_grid = grid_cells
+        
+
+    def generate_flight_paths_points(self):
+        ''' For each of the boxes allocated to the operator, get the centroid and buffer to get a flight path. A 75 m radius is provided to have flight paths within each of the boxes '''
+        # Iterate over the flight_grid
+        for grid_cell in self.flight_grid:
+            center = grid_cell.centroid
+            ## Transfrom to buffer 100 m diameter circle on which the drone will fly            
+            transformer = Transformer.from_crs("epsg:4326", "epsg:3857")
+            transformed_x,transformed_y = transformer.transform(center.x, center.y)
+            pt = Point(transformed_x,transformed_y)
+            buffer = pt.buffer(70) # build a buffer so that the radius is 70m for the track
+            buffer_points =  zip(buffer.exterior.coords.xy[0], buffer.exterior.coords.xy[1])
+            proj_buffer_points = []
+            # reproject back to ESPG 4326
+            transformer2 = Transformer.from_crs("epsg:3857", "epsg:4326")
+            for point in buffer_points:
+                x = point[0]
+                y = point[1]
+                x, y = transformer2.transform(x, y)
+                proj_buffer_points.append((x, y))
+            buffered_path = Polygon(proj_buffer_points)
+            altitude = 70.0
+            flight_points_with_altitude = []
+            x, y = buffered_path.exterior.coords.xy
+            for coord in range(0,len(x)):
+                flight_points_with_altitude.append((x[coord], y[coord], altitude))
+
+            # Build a list of points so that they can be fed to the sim and outputted. 
+            self.flight_points.append(flight_points_with_altitude)
+
+
+class TrackWriter():
+    ''' A class to write data geometry as files in geographical data format''' 
+
+    def __init__(self, path_points, bboxes) -> None:
+
+        self.flight_path_points = path_points
+        self.bboxes = bboxes
+        self.loop_counter = 0
+        
+        output_directory = Path('outputs').mkdir(parents=True, exist_ok=True) # Create outputs directory if it does not exist
```

Comment:
`outputs` is a little vague (and its contents will actually be inputs to aircraft_state_replayer and display_data_evaluator); could we output to `test_definitions/{NAME_OF_PARTICULAR_DEFINITION}`?  For instance,  the current defaults would probably write to `test_definitions/che`.

Reply:
Understood, I have now fixed it with the country code, we may need a different administration levels for e.g. for large countries perhaps states, civil aviation zones (?) . For the moment, it is OK.